ena-submission/scripts/test_ena_submission_integration.py

```diff
@@ -297,20 +322,22 @@ def check_project_submission_has_errors(
 
 
 def set_db_to_known_erz_accession(
-    db_config: SimpleConnectionPool, sequences_to_upload: dict[str, Any]
+    db_config: SimpleConnectionPool, sequences_to_upload: dict[str, Any], single_segment: bool
 ) -> None:
     """Sets erz-accession to known public accessions"""
     for full_accession, data in sequences_to_upload.items():
         accession, version = full_accession.split(".")
         organism = data["organism"]
         if organism == "cchf":
+            segment_order = ["L"] if single_segment else ["L", "M"]
+            erz_accession = "ERZ24985816" if single_segment else "ERZ24784470"
```

Comment:
Should we use `ERZ24908522` here as well as?

Reply:
I'm confused - this doesn't return a GCA? maybe we should add a comment here

Reply:
Yeah I found only one single-segment submission we've ever done (on the non-broker account) and that one doesn't have a GCA!

Reply:
You can use `ERZ24908522` (it also goes back to your testing and should return one segment) - but as this tests the WAITING case its probably good to do this - maybe just add a comment :-)

Reply:
Wait, ERZ24908522 never had an assembly submitted? I can't see the ERZ at all in processed.

Reply:
oh it appears this is an example we submitted on the broker account... which is weird as the tests are definitely on the test account

Reply:
omg... the tests are only checking submission of cchf... well this is great that we figured this out!!! @corneliusroemer thanks so much for flagging this!!!!